packages/server/tests/acceptance/rpc_batch2.spec.ts

```diff
@@ -726,7 +726,7 @@ describe('@api-batch-2 RPC Server Acceptance Tests', function () {
     async function createNftHTSToken(account) {
       const mainContract = new ethers.Contract(mainContractAddress, TokenCreateJson.abi, accounts[0].wallet);
       const tx = await mainContract.createNonFungibleTokenPublic(account.wallet.address, {
-        value: BigInt('10000000000000000000'),
+        value: BigInt('30000000000000000000'),
```

Comment:
For literals, might be clear to use the `BigInt` `n` suffix, _e.g._

```suggestion
        value: 30000000000000000000n,
```